docs/tutorials/upgrading-airbyte.md

```diff
@@ -40,10 +40,10 @@ On the other hand, if you don't mind losing your current Airbyte configuration o
 Here's an example of what might look like with the values filled in. It assumes that the downloaded `airbyte_archive.tar.gz` is in `/tmp`.
 
 ```bash
-docker run --rm -v /tmp:/config airbyte/migration:0.17.2-alpha --\
+docker run --rm -v /tmp:/config airbyte/migration:0.17.3-alpha --\
   --input /config/airbyte_archive.tar.gz\
   --output /config/airbyte_archive_migrated.tar.gz\
-  --target-version 0.17.2-alpha
+  --target-version 0.17.3-alpha
```

Comment:
This should be `0.17.0-alpha` so users don't get wrong target version exceptions.

I've tweaked the docs/migration in this PR: https://github.com/airbytehq/airbyte/pull/2583 for future releases

Reply:
fixed.